design-system/react/src/components/CardList/CardListItem.tsx

```diff
@@ -37,8 +37,7 @@ const styles = {
     layer: 'layer-card',
     position: 'relative',
     overflow: 'hidden',
-    py: '$3',
-    px: '$4',
+    padding: '$4 !important',
```

Comment:
really need this `!important`?

Reply:
Isn't working as expected, need to digging into stitches to see why, but I just let like this to avoid do this on this pull request @LuizAsFight